windows_containers/windows-containers-release-notes-7-x.adoc

```diff
@@ -28,6 +28,25 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-7-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 7.1.0
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 7.1.0 were released in link:https: //access.redhat.com/errata/RHBA-2023:NEED[RHBA-2023:NEED].
+
+[IMPORTANT]
+====
+Due to a known issue, your {product-title} cluster must be on version 4.12.3 or greater before updating the WMCO from version 7.0.1 to version 7.1.0.
+====
+
+[id="wmco-7-1-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `containerd` container runtime reported an incorrect version on each Windows node because repository tags were not propagated to the build system. This configuration caused `containerd` to report its Go build version as the version of each Windows node. With this update, the correct version is injected into the binary during build time, so that `containerd` reports the correct version for each Windows node. (link:https://issues.redhat.com/browse/OCPBUGS-7843[**OCPBUGS-7843**])
+
+* Previously, the Windows Machine Config Operator (WMCO) could not drain daemon set workloads. This issue caused Windows daemon set pods to block Windows nodes that the WMCO attempted to remove or upgrade. With this update, the WMCO includes additional role-based access control (RBAC) permissions, so that the WMCO can remove daemon set workloads. The WMCO can also delete any processes that were created with the containerd shim, so that daemonset containers do not exist on a Windows instance after a WMCO removes a node from a cluster. link:https://issues.redhat.com/browse/OCPBUGS-8056[*OCPBUGS-8056*])
+
+* Previously, on an Azure Windows Server 2019 platform that does not have Azure container services installed, WMCO would fail to deploy Windows instances and would display the _Install-WindowsFeature : Win32 internal error "Access is denied" 0x5 occurred while reading the console output buffer_ error message. The failure occurred because the Microsoft `Install-WindowsFeature` cmdlet displays a progress bar, which cannot be sent over an SSH connection. This fix hides the progress bar. As a result, Windows instances can be deployed as nodes. (link:https://issues.redhat.com/browse/OCPBUGS-14445[*OCPBUGS-14445*])
```

Comment:
```suggestion
* Previously, on an Azure Windows Server 2019 platform that does not have Azure container services installed, WMCO would fail to deploy Windows instances and would display the error message `Install-WindowsFeature : Win32 internal error "Access is denied" 0x5 occurred while reading the console output buffer`. The failure occurred because the Microsoft `Install-WindowsFeature` cmdlet displays a progress bar, which cannot be sent over an SSH connection. This fix hides the progress bar. As a result, Windows instances can be deployed as nodes. (link:https://issues.redhat.com/browse/OCPBUGS-14445[*OCPBUGS-14445*])
```

Reply:
Did you mean to put this error in backticks instead of italics? Jw because it seems that it is still in italics. I would also suggest backticks for error messages that appear in the console. Usually I reserve italics for book references to an Additional Resource

Reply:
Not sure if you meant me @skrthomas, but yeah I was going for just backticks.

Reply:
LOL sorry, should've specified _who_ I was talking to, but I was asking if @mburke5678 meant to keep the italics or use backticks.